src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -58,7 +58,13 @@ angular.module('ngWig')
           return;
         }
 
-        let pasteContent = (event.originalEvent || event).clipboardData.getData('text/plain');
+        let pasteContent;
+        if (window.clipboardData && window.clipboardData.getData) { // IE  
```

Comment:
Angular UI also had this problem and they fixed it in [this](https://github.com/angular-ui/ui-select/issues/789) way. So, I guess it shall be ok. I have also tested it with [this](https://plnkr.co/edit/J4OIhggQYzJ5VJNn7iLf?p=preview) plunker